src/main/java/seedu/address/logic/commands/AddAppointmentCommand.java

```diff
@@ -76,8 +78,12 @@ public CommandResult execute(Model model) throws CommandException {
                 .filter(person -> person instanceof Doctor && person.getName().equals(doctorName))
                 .findAny();
 
-        Person patientToEdit = patient.get();
-        Person doctorToEdit = doctor.get();
+        Person doctorToEdit = doctor.orElseThrow(() ->
+                new CommandException(MESSAGE_INVALID_DOCTOR_DISPLAYED_NAME + doctorName));
+
+        Person patientToEdit = patient.orElseThrow(() ->
+                new CommandException(MESSAGE_INVALID_PATIENT_DISPLAYED_NAME + patientName));
```

Comment:
great use of the methods in the Java Optional library to use exceptions and handle errors gracefully if the doctor/patient aren't there